TestProjects/UniversalGraphicsTest_Lighting/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -86,15 +83,6 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/Scenes/146_AdditionalLightShadowCustomResolution.unity
     guid: 5cb07294042344649a35ad91c91b539a
-  - enabled: 0
-    path: Assets/Scenes/154_ReflectionProbes_Blending.unity
-    guid: 563a251f782be254a91472c4db94e506
-  - enabled: 0
-    path: Assets/Scenes/155_ReflectionProbes_BoxProjection.unity
-    guid: dbc896364427cae40affa52a983b43a2
```

Comment:
These were removed automatically, I guess there might be some tooling removing disabled scenes from editor build settings? 
 
 I plan to move these scenes to _Foundation project in follow up PR anyway.